tests/vercelConfig.test.ts

```diff
@@ -1,10 +1,12 @@
-process.env.NEXT_PUBLIC_AXIOM_URL = undefined;
-process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT = 'https://api.axiom.co/v1/integrations/vercel';
-
-import { test, expect } from '@jest/globals';
+import { test, expect, vi } from 'vitest';
 import { config } from '../src/config';
 import { EndpointType } from '../src/shared';
 
+vi.hoisted(() => {
+  process.env.NEXT_PUBLIC_AXIOM_URL = undefined;
+  process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT = 'https://api.axiom.co/v1/integrations/vercel';
+});
+
 test('reading vercel ingest endpoint', () => {
   let url = config.getIngestURL(EndpointType.webVitals);
```

Comment:
Feels like a lot of the tests are testing the `config.getIngestURL` with different parameters passed to it. Would it make sense to make a file to test the function itself instead of a file for each case?

Reply:
the thing is, the environment variables have to be set before importing the logger files. For different scenarios we need different set of environment variables. Not sure If I can change the values without re-importing the files.